collectors/wallet_collector.go

```diff
@@ -39,6 +45,8 @@ type WalletCollector struct {
 func NewWalletCollector(lnd *lndclient.LndServices,
 	errChan chan<- error) *WalletCollector {
 
+	// these labels are specific for key_count metrics
+	keyCountLabels := []string{"account_name", "address_type"}
```

Comment:
Could `internal` and `external` be additional labels as well? Rather than needing to split out into two separate metrics? Since they seem functionally the same except for internal/external differentiation.

Reply:
Yes, that makes sense, will change it

Reply:
How would you call the label? I can think of `visibility` or `context` or `external={true|false}` ?

Reply:
`key_type` or `key_branch`?

Reply:
`key_type` it is